lib/clients/api-client.js

```diff
@@ -70,8 +70,12 @@ SlackAPIClient.prototype.send = function(method, options, callback) {
       if (Array.isArray(options.attachments)) {
         options.attachments = JSON.stringify(options.attachments);
       }
-      requestOptions.method = 'GET';
-      requestOptions.qs = options;
+
+      // We default to a POST request here because long messages will exceed the ~2000 character
+      // limit in the url for a GET request.
+      requestOptions.method = 'POST';
+      requestOptions.headers = {'Content-Type': 'application/x-www-form-urlencoded'};
+      requestOptions.body = querystring.stringify(options);
```

Comment:
Just confirming that you've tested this on the Slack API and you're sure Slack still wants the body in the querystring format and not JSON or other?